examples/integrations/oci/atp-cdi/src/main/java/io/helidon/examples/integrations/oci/atp/cdi/AtpResource.java

```diff
@@ -69,21 +73,29 @@ public Response generateWallet() {
 
         GenerateAutonomousDatabaseWallet.Response response = entity.get();
 
-        StreamingOutput stream = output -> response.writeTo(Channels.newChannel(output));
-
-        Response.ResponseBuilder ok = Response.ok(stream, MediaType.APPLICATION_OCTET_STREAM_TYPE)
-                .header("opc-request-id", ociResponse.headers().first("opc-request-id").orElse(""))
-                .header("request-id", ociResponse.requestId());
-
-        ociResponse.headers()
-                .first(Http.Header.CONTENT_TYPE)
-                .ifPresent(ok::type);
-
-        ociResponse.headers()
-                .first(Http.Header.CONTENT_LENGTH)
-                .ifPresent(it -> ok.header(Http.Header.CONTENT_LENGTH, it));
+        try {
```

Comment:
Consider try-with-resources instead so that the `close()` is effectively put into a `finally` block for you

Reply:
Changing examples a lot in upcoming PR to add how to use wallet and do DB operations using it. Will take this up with that.